.github/workflows/test.yml

```diff
@@ -73,4 +73,6 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: backend-coverage
-      - uses: codecov/codecov-action@v3
+      - name: (TODO) Upload coverage reports
+        run: |
+          echo "Coverage files: .coverage, coverage, coverage.xml"
```

Comment:
This is fine, but thinking through other options:

1. Re-add coveralls (we were using that before, and removed it in https://github.com/oddbird/Metecho/pull/2149). I think David said that's what SFDO uses?
2. Re-add local coverage checks (also removed in https://github.com/oddbird/Metecho/pull/2149).

Reply:
Both fixed in 19ce97b357f169ec48025faf35c9a97d782c3a4a and the previous commits

![image](https://user-images.githubusercontent.com/2391102/168873520-4e838b40-dbaa-4267-a202-870addbb1c13.png)

I've noticed the Python and JS coverage data are treated as separate builds in the UI, not sure if that's a problem

Reply:
Yeah, [before](https://github.com/oddbird/Metecho/pull/2149/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L129-L153) we were using Coveralls support for [Parallel Builds](https://docs.coveralls.io/parallel-build-webhook) to merge the results into one build. It might be worth doing that again?